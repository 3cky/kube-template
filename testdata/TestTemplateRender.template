plain text: the quick brown fox jumps over the lazy dog

helper math functions: 2+2={{add 2 2}}, 2-2={{sub 2 2}}, 2*2={{mul 2 2}}, 2/2={{div 2 2}}

helper string functions: {{ "toUpper" | toUpper }}, {{ "toLower" | toLower }}, {{ "upper" | upper }}, {{ "LOWER" | lower }}

pod names: {{range pods}}{{.Name}}{{end}}
