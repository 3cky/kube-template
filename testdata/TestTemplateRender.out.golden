plain text: the quick brown fox jumps over the lazy dog

helper math functions: 2+2=4, 2-2=0, 2*2=4, 2/2=1

helper string functions: TOUPPER, tolower, UPPER, lower

pod names: pod1
